src/rules/escape-yaml-special-characters.ts

```diff
@@ -1,11 +1,11 @@
 import {Options, RuleType} from '../rules';
 import RuleBuilder, {BooleanOptionBuilder, ExampleBuilder, OptionBuilderBase} from './rule-builder';
 import dedent from 'ts-dedent';
-import {escapeStringIfNecessaryAndPossible, formatYAML} from '../utils/yaml';
+import {escapeStringIfNecessaryAndPossible, formatYAML, QuoteCharacter as QuoteCharacter} from '../utils/yaml';
```

Comment:
Why are we including as here and not in other imports?

Reply:
No reason, it is a leftover of the other refactoring. I'll fix it